pom.xml

```
@@ -31,7 +31,7 @@
     <scm>
         <connection>scm:git:https://gitbox.apache.org/repos/asf/cxf.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/cxf.git</developerConnection>
-        <url>https://gitbox.apache.org/repos/asf?p=cxf.git;a=summary</url>
+        <url>https://gitbox.apache.org/repos/asf/cxf.git</url>
```

I think `;a=summary` is confusing, could you please change it to:

```suggestion
        <url>https://gitbox.apache.org/repos/asf?p=cxf.git</url>
```

Thank you.